packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -63,26 +63,26 @@
     "simple-react-validator": "^1.6.1"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/build-tools": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-bentley": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
-    "@itwin/core-i18n": "^3.0.0",
-    "@itwin/core-markup": "^3.0.0",
-    "@itwin/core-orbitgt": "^3.0.0",
-    "@itwin/core-quantity": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/imodel-components-react": "^3.0.0",
-    "@itwin/presentation-common": "^3.0.0",
-    "@itwin/presentation-components": "^3.0.0",
-    "@itwin/presentation-frontend": "^3.0.0",
-    "@itwin/webgl-compatibility": "^3.0.0",
+    "@itwin/appui-abstract": "^3.2.0",
+    "@itwin/appui-layout-react": "^3.2.0",
+    "@itwin/appui-react": "^3.2.0",
+    "@itwin/build-tools": "^3.2.0",
+    "@itwin/components-react": "^3.2.0",
+    "@itwin/core-bentley": "^3.2.0",
+    "@itwin/core-common": "^3.2.0",
+    "@itwin/core-frontend": "^3.2.0",
+    "@itwin/core-geometry": "^3.2.0",
+    "@itwin/core-i18n": "^3.2.0",
+    "@itwin/core-markup": "^3.2.0",
+    "@itwin/core-orbitgt": "^3.2.0",
+    "@itwin/core-quantity": "^3.2.0",
+    "@itwin/core-react": "^3.2.0",
+    "@itwin/eslint-plugin": "^3.2.0",
+    "@itwin/imodel-components-react": "^3.2.0",
+    "@itwin/presentation-common": "^3.2.0",
+    "@itwin/presentation-components": "^3.2.0",
+    "@itwin/presentation-frontend": "^3.2.0",
+    "@itwin/webgl-compatibility": "^3.2.0",
```

Comment:
@aruniverse I'm bumping the required version up to 3.2.0 due to some needed changes required by the widget. What do you think?

Reply:
thats fine with me, typically when you bump the peer deps its a breaking change, but since its 0.x just change the change type to minor not patch